Sources/ClientRuntime/Networking/Http/URLSession/FoundationStreamBridge.swift

```diff
@@ -19,20 +19,26 @@ import class Foundation.Timer
 import struct Foundation.TimeInterval
 import protocol Foundation.StreamDelegate
 
-/// Reads data from a smithy-swift native `ReadableStream` and streams the data to a Foundation `InputStream`.
+/// Reads data from a smithy-swift native `ReadableStream` and streams the data through to a Foundation `InputStream`.
+///
+/// A pair of Foundation "bound streams" is created.  Data from the `ReadableStream` is transferred into the Foundation
+/// `OutputStream` until the `ReadableStream` is closed and all data has been read from it.  The Foundation
```

Comment:
Q: Line 22 says data is read from `ReadableStream` to a `InputStream`. Line 24&25 say data is read from `ReadableStream` to `OutputStream`. So is `OutputStream` a stepping stone (e.g., `ReadableStream` => `OutputStream` => `InputStream`)? If so, why does there need to be one?

Reply:
Line 22 is supposed to summarize where the data comes in to the `FoundationStreamBridge` and where it goes out.  Line 24/25 are individual steps within the `FoundationStreamBridge`.

Overall, data flows like this:
```
Customer events or data > ClientRuntime.ReadableStream > FoundationStreamBridge buffer > Foundation.OutputStream > bound stream buffer > Foundation.InputStream > URLRequest streaming body
```

We need the `OutputStream` because a pair of bound streams is how you get data through to the `InputStream` that we need for streaming our URLRequest body.  Put another way, we can't put data into an `InputStream` unless we do so with a bound `OutputStream`.